tests/queries/0_stateless/02707_protobuf_unnamed_tuple_as_nested_message.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+echo 'syntax = "proto3";
+
+message Nested {
+    int32 a = 1;
+    string b = 2;
+    repeated int32 c = 3;
+};
+
+message Message {
+    Nested x = 1;
+};' > 02707_schema_$CLICKHOUSE_TEST_UNIQUE_NAME.proto
```

Comment:
You can just put a ready *.proto file to `/tests/queries/0_stateless/format_schemas` with other test format schemas.